api_specifications/note-api-0.1.0.yaml

```diff
@@ -0,0 +1,256 @@
+        description: Search criteria + RequestInfo meta data.
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/NoteSearchRequest'
+        required: true
+      responses:
+        '200':
+          description: ResponseInfo with note matching search criteria
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/NoteListResponse'
+        '400':
+          description: Note search failed. Refer to response body for specifics.
+          content:
+            '*/*':
+              schema:
+                $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+  /element/v1/search:
+    post:
+      tags:
+        - Element
+      summary: Search for constitute elements by case type
+      requestBody:
+        description: Search criteria + RequestInfo meta data.
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/ConstituteElementSearchRequest'
+        required: true
+      responses:
+        '200':
+          description: ResponseInfo with element matching search criteria
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ConstituteElementListResponse'
+        '400':
+          description: Constitute Element search failed. Refer to response body for specifics.
+          content:
+            '*/*':
+              schema:
+                $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+components:
+  schemas:
+    Note:
+      description: Note registory
+      required:
+        - note
+        - caseId
+        - tenantId
+        - submittedById
+      type: object
+      properties:
+        id:
+          type: string
+        note:
+          type: string
+        caseId:
+          type: string
+        hearingId:
+          type: string
+        tenantId:
+          type: string
+          description: This is tenantId where the system is implemented.
+          minLength: 2
+          maxLength: 64
+        createdDate:
+          type: integer
+          format: int64
+        modifiedDate:
+          type: integer
+          format: int64
+        submittedById:
+          type: string
+        workflow:
+          $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/Workflow
+        additionalDetails:
+          type: object
+          description: This field is meant to hold any extra information that we want to store for a note.
+        auditDetails:
+          $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/AuditDetails
+        constituteElements:
+          type: array
```

Comment:
this is not a field inside note. we discussed. make it a separate registry

Reply:
ok